python/pyxbos/pyxbos/drivers/weather_prediction/parker.proto

```diff
@@ -0,0 +1,189 @@
+syntax = "proto3";
+package xbospb;
+import "nullabletypes.proto";
+message ParkerState {
```

Comment:
Can you remove the `.proto` file from the Python package? I like to keep all of them in a single standard place (`/proto`)